HenHen.Framework/Worlds/Mediums/Medium.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Affectionate Dove <contact@affectionatedove.com>.
+// Licensed under the Affectionate Dove Limited Code Viewing License.
+// See the LICENSE file in the repository root for full license text.
+
+using HenHen.Framework.Numerics;
+
+namespace HenHen.Framework.Worlds.Mediums
+{
+    public class Medium
```

Comment:
Override the ToString method so that for example it returns such a string:
`{Type=Ground,Triangle=TRIANGLESTRING}`
where TRIANGLESTRING is replaced with Triangle.ToString().

Reply:
See https://github.com/Affectionate-Dove/HenHen/blob/bfc43ec4abc1bc3182f79e6cde6e6578f46772ad/HenHen.Framework/Numerics/Triangle3.cs#L27 for reference